.azure-pipelines/resources/chaincode/commercialpaper/commercial-paper/paper.go

```diff
@@ -0,0 +1,138 @@
+// Copyright the Hyperledger Fabric contributors. All rights reserved.
```

Comment:
seeing as we're manually copying a version of this into resources, rather than cloning it, or some other way to automagically get it, are we ok with it falling out of step with commercial paper in fabric-samples?

Reply:
It should be fine to fall out of line with it. The integration tool will likely update with it but as long as the chaincode gives the answers it expects there isn't really any reason to have the same code.